helm/charts/oathkeeper/values.yaml

```diff
@@ -29,6 +29,10 @@ sidecar:
     tag: v0.1.2
   envs: {}
 
+# -- Pod priority
+# https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+priorityClassName: ""
```

Comment:
also please update the values.yaml files in the `hack/values` directory to set those fields. This allows the CI to verify if they generate valid manifest files